tasks/helpers/responsehelper.js

```diff
@@ -0,0 +1,16 @@
+const responseHelper = `
+    /**
+     * @static
+     * @param {response}  response object
+     * @param {message}  message
+     * @param {data}  data
+     * @param {status}  status code
+     * @returns {object} object
+    */
+    export default (res, message, data, status, data) => {
+      res.status(status).json({
+        message,
```

Comment:
At the moment I could just suggest we go with 
```javascript
res.status(status).send({message, status, data});
```
Instead of making use of `.json`. That can be the only request on this part.